autoload/test/python/pytest.vim

```diff
@@ -44,6 +44,10 @@ function! test#python#pytest#executable() abort
     let pipenv_prefix = "pipenv run "
   endif
 
+  if filereadable("poetry.lock")
```

Comment:
Could this be an `elseif` chained on the previous conditional? That would be more intention revealing, because either Pipenv or Poetry will be used, never both.